### PR TITLE
Removed broken link

### PR DESCRIPTION
**Pull request by n-boshnakov**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
This PR removes a link to a file which no longer exists (it was removed due to being outdated).

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

File: docs/extensions/logging-and-monitoring.md [hunk 1]
```diff
@@ -156,4 +156,3 @@ If the type of logs exposed in the Plutono instances needs to be changed, it is
 
 * [GitHub Issue Describing the Concept](https://github.com/gardener/gardener/issues/1351)
 * [Exemplary Implementation (Monitoring) for the GCP Provider](https://github.com/gardener/gardener-extension-provider-gcp/blob/master/charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-observability.yaml)
-* [Exemplary Implementation (Logging) for the OpenStack Provider](https://github.com/gardener/gardener-extension-provider-openstack/blob/master/charts/gardener-extension-provider-openstack/templates/configmap-logging.yaml)
```
Comment (shafeeqes) [inline docs/extensions/logging-and-monitoring.md:159]: Can we link https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterfilters-logging.yaml and https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterparsers-logging.yaml instead?

Comment (n-boshnakov) [reply docs/extensions/logging-and-monitoring.md:159]: Of course! What would you like these to be called in the references section?

Comment (shafeeqes) [reply docs/extensions/logging-and-monitoring.md:159]: Maybe Exemplary Implementation (ClusterFilter), Exemplary Implementation (ClusterParser) ?

**Review by shafeeqes — APPROVED** [review]
Thanks!
/lgtm

**Pull request merged**